package.json

```diff
@@ -17,33 +17,37 @@
     "url": "https://github.com/TwilioDevEd/appointment-reminders-node"
   },
   "keywords": [
-   "node",
-   "appointment reminders",
-   "twilio",
-   "express",
-   "sms",
-   "notification"
+    "node",
+    "appointment reminders",
+    "twilio",
+    "express",
+    "sms",
+    "notification"
   ],
-  "engines" : { "node" : ">=4.1.0 <5.5.0" },
+  "engines": {
+    "node": ">=4.1.0 <5.5.0"
+  },
   "dependencies": {
     "bluebird": "^3.3.1",
-    "body-parser": "~1.13.2",
-    "cookie-parser": "~1.3.5",
+    "body-parser": "~1.15.2",
+    "cookie-parser": "~1.4.3",
     "cron": "^1.1.0",
-    "debug": "~2.2.0",
-    "express": "~4.13.1",
+    "debug": "~2.3.0",
+    "express": "~4.14.0",
     "jade": "~1.11.0",
     "moment": "^2.11.2",
     "moment-timezone": "^0.5.0",
     "mongoose": "^4.4.3",
-    "morgan": "~1.6.1",
+    "morgan": "~1.7.0",
     "serve-favicon": "~2.3.0",
-    "twilio": "^3.3.0-edge"
+    "twilio": "3.0.0-rc.13"
```

Comment:
is it missing an `^` :grey_question: 
Because like that it will be always anchored to a specific version, and the previous one used to have the `^` on  `"^3.3.0-edge"`